retention/src/test/java/org/apache/falcon/retention/FeedEvictorTest.java

```diff
@@ -462,6 +462,43 @@ public void testFeedBasePathExists() throws Exception {
         }
     }
 
+    @Test
+    public void testEvictionStatsMetaWithNoPattern() throws Exception {
+        try {
+            Configuration conf = cluster.getConf();
+            FileSystem fs = FileSystem.get(conf);
+            fs.delete(new Path("/"), true);
```

Comment:
@vramachan that explains why these tests did not fail in the first place. Thanks for the patch. +1 and will commit soon